src/ucp/tag/rndv.c

```diff
@@ -462,7 +462,6 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_rndv_progress_rma_get_zcopy, (self),
     ucp_lane_index_t lane;
 
     ucp_rndv_get_lanes_count(rndv_req);
-    ucs_assert_always(rndv_req->send.rndv_get.lane_count > 0);
```

Comment:
maybe move it below the `if` handling fallback to AM?

Reply:
ok